osquery/logger/logger.cpp

```diff
@@ -452,14 +452,16 @@ void BufferedLogSink::WaitTillSent() {
     self.senders.pop();
   }
 
-  first.wait();
+  if (!isPlatform(PlatformType::TYPE_WINDOWS)) {
+    first.wait();
+  }
 }
 
 Status LoggerPlugin::call(const PluginRequest& request,
                           PluginResponse& response) {
   if (FLAGS_logger_secondary_status_only &&
       !BufferedLogSink::isPrimaryLogger(getName()) &&
-      (request.count("string") || request.count("snapshot"))) {
+      (request.count("string") || request.count("snapsot"))) {
```

Comment:
We lost an `h` here? Was this a fat-finger? Or do we really want to cont `snapsot`?